app/[locale]/error.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import type { FC } from 'react';
+
+const ErrorPage: FC<{ error: Error }> = ({ error }) => (
+  <div className="container">
+    <h2>500: Internal Server Error</h2>
+    <h3>This Page has thrown a non-recoverable Error</h3>
+    <small>
+      Details: <br />
+      <pre>
+        <code>${error.message}</code>
```

Comment:
from an information security standpoint, I tell my teams that it isn't great to provide end-users with details as to what went wrong. We'll already be getting the reports afterall.

This might not be so bad because it's only `error.message` and not the whole thing or stacktrace (which I see people do and that's way worse)